### ISSUE TITLE
[Feature Request] Add “Horizontal Margin” to the panel settings

### ISSUE THREAD

**Issue by Quasar-0330** [nwg-piotr/nwg-panel#331]
Currently the panel settings only include margin top and margin-bottom, and the left and right margins are
```
#panel-top {
    margin: 0 4px;
}
```
and the left and right margins cannot be changed even if they are set in css.
(There may be some discomfort due to the use of translation software)

**Comment by nwg-piotr** [comment]
Yes, we are on gtk-layer-shell: https://github.com/nwg-piotr/nwg-panel/blob/fd0bbf7c325724e8e80f893dfd6f80a59590ea4c/nwg_panel/main.py#L827

Sure left and right margin could be added. I'll take a look at this later tonight.

**Comment by nwg-piotr** [comment]
Sorry, I forgot what this code looked like. We can not define side margins, as we set size request to whole the window. Instead, uncheck "auto" and set the panel width manually.

![obraz](https://github.com/user-attachments/assets/ed6e00f9-57bc-430c-b189-79f14a8a5996)
